typedoc.json

```diff
@@ -1,11 +1,9 @@
 {
-  "inputFiles": ["lib/"],
+  "entryPoints": ["lib/"],
```

Comment:
these options have since been removed in version 0.20 as stated in the release [notes](https://github.com/TypeStrong/typedoc/releases/tag/v0.20.0)

Reply:
Nice catch